beets/mediafile.py

```diff
@@ -2005,6 +2005,62 @@ def update(self, dict):
         out_type=float,
     )
 
+    # EBU R128 fields.
+    r128_track_gain = MediaField(
+        MP3DescStorageStyle(
+            u'R128_TRACK_GAIN'
+        ),
+        MP3DescStorageStyle(
+            u'r128_track_gain'
+        ),
+        MP3SoundCheckStorageStyle(
+            key='COMM',
+            index=0, desc=u'iTunR128',
+            id3_lang='eng'
+        ),
+        MP4StorageStyle(
+            '----:com.apple.iTunes:r128_track_gain'
+        ),
+        MP4SoundCheckStorageStyle(
+            '----:com.apple.iTunes:iTunR128',
+            index=0
+        ),
```

Comment:
I'd recommend dropping the SoundCheck storage styles (this and the MP3 variant above). The SC format is included for the RG tags for compatibility with the iTunes-specific normalization metadata, which we should probably (eventually) break off as a *third* standard that deserves a third set of tags.

We can also cross our fingers and hope that this solves the crash on Python 3…